stripe/src/test/java/com/stripe/android/view/CardInputTestActivity.java

```diff
@@ -24,14 +20,21 @@
     public static final String VALID_VISA_WITH_SPACES = "4242 4242 4242 4242";
 
     private CardInputWidget mCardInputWidget;
+    private CardMultilineWidget mCardMultilineWidget;
+    private CardMultilineWidget mNoZipCardMulitlineWidget;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        setTheme(R.style.Theme_AppCompat);
```

Comment:
The `CardMultilineWidget` can't be added to a class that doesn't use an AppCompat theme.